src/features/auth/lib/authMachine.ts

```diff
@@ -169,6 +169,8 @@ export type BlockchainState = {
 
 const API_URL = CONFIG.API_URL;
 
+export type MachineState = State<Context, BlockchainEvent, BlockchainState>;
```

Comment:
I think its better to call it AuthMachineState, to prevent misuse of that type when we want to get state from game or goblin machine.

Reply:
Addressed